src/AerospikeCache.php

```diff
@@ -0,0 +1,133 @@
+<?php
+declare(strict_types=1);
+
+namespace Lmc\AerospikeCache;
+
+use Aerospike;
+use Symfony\Component\Cache\Adapter\AbstractAdapter;
+use Symfony\Component\Cache\Exception\CacheException;
+
+class AerospikeCache extends AbstractAdapter
+{
+    private const WRAPPER_NAME = 'data';
+
+    /** @var Aerospike */
+    private $aerospike;
+
+    /** @var string $aerospikeNamespace */
+    private $namespace;
+
+    /** @var string $set */
+    private $set;
+
+    public function __construct(
+        \Aerospike $aerospike,
+        string $namespace,
+        string $set = 'cache',
+        string $cacheNamespace = '',
+        int $defaultLifetime = 0
+    ) {
+        $this->aerospike = $aerospike;
+        $this->namespace = $namespace;
+        $this->set = $set;
+        parent::__construct($cacheNamespace, $defaultLifetime);
+    }
+
+    protected function doFetch(array $ids): array
+    {
+        $result = [];
+
+        $keys = $this->initializeKeysForAerospike($ids);
+
+        $this->aerospike->getMany($keys, $records);
+
+        foreach ($records as $record) {
+            if ($record['metadata'] !== null) {
+                $result[$record['key']['key']] = isset($record['bins'][self::WRAPPER_NAME]) ?
+                    $record['bins'][self::WRAPPER_NAME] : null;
+            }
+        }
+
+        return $result;
+    }
+
+    private function initializeKeysForAerospike(array $ids): array
+    {
+        return array_reduce(
+            $ids,
+            function (array $values, $id) {
+                $values[] = $this->createKey($id);
+
+                return $values;
+            },
+            []
+        );
+    }
+
+    protected function doHave($id)
+    {
+        $statusCode = $this->aerospike->get($this->createKey($id), $read);
```

Comment:
`$read` is not used, since it should not be used here, it could be _ignored_ explicitly by naming it something like `$ignoredRecord` or `$_`

Reply:
changed to ignoredRecord

Reply:
and will you push that change?

Reply:
Yeah sorry. Also rewrote it a bit so now it is just one line
return \Aerospike::OK === $this->aerospike->get($this->createKey($id), $ignoredRecord);